api/ca.proto

```diff
@@ -43,6 +43,10 @@ message IssueNodeCertificateRequest {
 	// Token represents a user-provided string that is necessary for new
 	// nodes to join the cluster
 	string token = 3;
+
+	// Spec is the spec of the node being requested, not all fields in
+	// spec will be honored.
```

Comment:
Thanks @aaronlehmann. Didn't see this one when updating the PR minutes ago. Will update the PR again shortly.